src/components/subscriptions/licenses/LicenseManagementModals/LicenseManagementRevokeModal.jsx

```diff
@@ -62,6 +62,18 @@ const generateRevokeModalSubmitLabel = (revokeAllUsers, userCount, totalToRevoke
   };
 };
 
+const checkIfFiltersPresent = (filtersAndValues) => {
```

Comment:
nit: this could likely be simplified as

```jsx
const checkIfFiltersPresent = filtersAndValues => filtersAndValues.some(filter => !!filter.filterValue);
```